adapters/Nend/NendAdapter/nativeads/GADMAdapterNendNativeVideoAd.m

```diff
@@ -0,0 +1,169 @@
+//
+//  GADMAdapterNendNativeVideoAd.m
+//  NendAdapter
+//
+//  Copyright © 2019 FAN Communications. All rights reserved.
+//
+
+#import "GADMAdapterNendNativeVideoAd.h"
+#import "GADMAdapterNendNativeAdLoader.h"
+
+@interface GADMAdapterNendNativeVideoAd () <NADNativeVideoDelegate, NADNativeVideoViewDelegate>
+
+@property(nonatomic, strong) NADNativeVideo *videoAd;
+@property(nonatomic, strong) NADNativeVideoView *nendMediaView;
+@property(nonatomic, strong) GADNativeAdImage *mappedIcon;
+@property(nonatomic, strong) NSDecimalNumber *userRating;
+
+@end
+
+@implementation GADMAdapterNendNativeVideoAd
```

Comment:
This class is missing `mediaContentAspectRatio`

https://developers.google.com/ad-manager/mobile-ads-sdk/ios/api/reference/Protocols/GADMediatedUnifiedNativeAd.html#mediacontentaspectratio

Reply:
I'll add mediaContentAspectRatio getter.